test/identifier/single-statement.spec.ts

```diff
@@ -522,6 +522,143 @@ describe('identifier', () => {
       });
     });
 
+    describe('identify SHOW statements', () => {
+      (['mysql', 'generic', 'mssql'] as Dialect[]).forEach((dialect) => {
+        it(`identify "SHOW KEYS" statements for ${dialect}`, () => {
```

Comment:
Can we use a parameterized setup for these tests given that their setup is basically the same?

```
[['KEYS', 'DATABASES', 'INDEX, 'TABLE_STATUS']].forEach((type) => {
  it(`identify "${type}" statements for ${dialect}`, () => {
    const sqlStatement = `SHOW ${type.split('_')};`;
    // rest of test stuff...
    const expected = [
      {
        start: 0,
        end: sqlStatement.length - 1,
        text: sqlStatement,
        type,
        executionType: 'INFORMATION',
        parameters: [],
      },
    ];

    expect(actual).to.eql(expected);
  });
});
```

This'll make adding new types of `SHOW` easier.